src/VscProcess.cpp

```diff
@@ -77,6 +77,30 @@ VscProcess::VscProcess() : myEStopState(0)
     }
   }
 
+  vsc_scm_target_set(vscInterface, 0);
+  vsc_scm_target_get(vscInterface);
+
+  // ros::Duration(0.1).sleep();
```

Comment:
Do we still need these commented out sleep statements? I'd go ahead and remove them if not.

Reply:
Good catch, will remove.